iri-orphans.cabal

```diff
@@ -0,0 +1,148 @@
+cabal-version: 1.18
+
+-- This file has been generated from package.yaml by hpack version 0.37.0.
+--
+-- see: https://github.com/sol/hpack
+
+name:           iri-orphans
+version:        0.0.0.0
+synopsis:       Orphan instances for iri
+description:    Orphan instances for types defined in the `iri` package.
+category:       Utils
+homepage:       https://github.com/freckle/iri-orphans#readme
+bug-reports:    https://github.com/freckle/iri-orphans/issues
+maintainer:     Freckle Education
+license:        MIT
+license-file:   LICENSE
+build-type:     Simple
+extra-source-files:
+    package.yaml
+extra-doc-files:
+    README.md
+    CHANGELOG.md
+
+source-repository head
+  type: git
+  location: https://github.com/freckle/iri-orphans
+
+library
+  exposed-modules:
+      Iri.Orphans
+      Iri.Orphans.Aeson
+      Iri.Orphans.Aeson.FromJSON
+      Iri.Orphans.Aeson.ToJSON
```

Comment:
A module per instance is a _bit_ extreme, IMO.

Reply:
I figure there's no way to import only one orphan from a module that defines two, so separating them and then collecting them into a reexporting module for convenience gives the user easy mode plus the ability to be more precise if they want.